t/14-isotovideo.t

```diff
@@ -130,47 +130,58 @@ subtest 'isotovideo with wheels' => sub {
-    checkout_wheels("$wheels_dir");
+    checkout_wheels($case_dir, $wheels_dir);
     is($repos[0], 'https://github.com/foo/bar.git', 'repo with full URL');
     is(scalar @repos, 1, 'one wheel');
     $specfile->spurt("version: v0.1\nwheels: [https://github.com/foo/bar.git#branch]");
-    checkout_wheels("$wheels_dir");
+    checkout_wheels($case_dir, $wheels_dir);
     is($repos[1], 'https://github.com/foo/bar.git#branch', 'repo URL with branch');
     is(scalar @repos, 2, 'one wheel');
     $specfile->spurt("version: v0.1\nwheels: [foo/bar]");
-    checkout_wheels("$wheels_dir");
+    checkout_wheels($case_dir, $wheels_dir);
     is(scalar @repos, 3, 'one wheel');
     is($repos[2], 'https://github.com/foo/bar.git', 'only wheel');
     $specfile->spurt("version: v0.1\nwheels: [foo/bar, spam/eggs]");
-    checkout_wheels("$wheels_dir");
+    checkout_wheels($case_dir, $wheels_dir);
     is($repos[4], 'https://github.com/spam/eggs.git', 'second wheel');
     is(scalar @repos, 5, 'two wheels');
     $specfile->remove;
-    is(checkout_wheels("$wheels_dir"), 1, 'no wheels');
+    is(checkout_wheels($case_dir, $wheels_dir), 1, 'no wheels');
     is(scalar @repos, 5, 'git never called');
 
     # also verify that isotovideo invokes the wheel code correctly
-    $bmwqemu::vars{CASEDIR} = "$data_dir/tests";
-    $specfile->spurt("version: v0.1\nwheels: [copy/writer]");
-    path($pool_dir, 'writer', 'lib', 'Copy', 'Writer')->make_path->child('Content.pm')->spurt("package Copy::Writer::Content; use Mojo::Base 'Exporter'; our \@EXPORT_OK = qw(write); sub write {}; 1");
-    path($pool_dir, 'writer', 'tests', 'pen')->make_path->child('ink.pm')->spurt("use Mojo::Base 'basetest'; use Copy::Writer::Content 'write'; sub run {}; 1");
-    my $log = combined_from { isotovideo(
-            opts => "wheels_dir=$wheels_dir casedir=$data_dir/tests schedule=pen/ink _exit_after_schedule=1") };
-    like $log, qr@Skipping to clone.+copy/writer@, 'already cloned wheel picked up';
-    like $log, qr/scheduling ink/, 'module from the wheel scheduled';
-    rmtree "$pool_dir/writer";
+    $bmwqemu::vars{CASEDIR} = $case_dir;
+
+    for my $dir (("$data_dir/wheels_dir", undef)) {
+        chdir $pool_dir;
+        $bmwqemu::vars{WHEELS_DIR} = $dir;
+        $wheels_dir = $dir // $pool_dir;
+
+        subtest "wheels in $wheels_dir" => sub {
+            $specfile->spurt("version: v0.1\nwheels: [copy/writer]");
+            path($wheels_dir, 'writer', 'lib', 'Copy', 'Writer')->make_path->child('Content.pm')->spurt("package Copy::Writer::Content; use Mojo::Base 'Exporter'; our \@EXPORT_OK = qw(write); sub write {}; 1");
+            path($wheels_dir, 'writer', 'tests', 'pen')->make_path->child('ink.pm')->spurt("use Mojo::Base 'basetest'; use Copy::Writer::Content 'write'; sub run {}; 1");
+            my $log = combined_from { isotovideo(
+                    opts => "wheels_dir=$wheels_dir casedir=$case_dir schedule=pen/ink _exit_after_schedule=1") };
```

Comment:
Here we always set `wheels_dir=$wheels_dir`, so it never tests the case where it's undef.
`$bmwqemu::vars{WHEELS_DIR}` is set in the test process, but isotovideo is running in its own process